roofit/roofitcore/inc/RooFuncWrapper.h

```diff
@@ -110,7 +131,7 @@ class RooFuncWrapper final : public RooAbsReal {
    {
       updateGradientVarBuffer();
 
-      _grad(_gradientVarBuffer.data(), out);
+      _grad(_gradientVarBuffer.data(), _observables.data(), out);
```

Comment:
Unrelated to this PR, but something I see in the lines below that should have been different to begin with:

Please use `getVal()` instead of `getValV()`. The `getValV()` function should not be called directly.